app/controllers/users/phone_setup_controller.rb

```diff
@@ -45,6 +48,10 @@ def set_setup_presenter
       )
     end
 
+    def setup_voice_preference?
+      params[:otp_delivery_preference].to_s == 'voice'
```

Comment:
The `.to_s` shouldn't be needed since the parameter would either be a string or `nil`, and in either case the result here would only be `true` if it were a string of `'voice'`.

```suggestion
      params[:otp_delivery_preference] == 'voice'
```